plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentAccuracyRadiusAttributeParser.kt

```diff
@@ -0,0 +1,29 @@
+package com.mapbox.maps.plugin.locationcomponent
+
+import android.content.Context
+import android.graphics.Color
+import android.util.AttributeSet
+
+/**
+ * Utility class for parsing [AttributeSet] to [LocationComponentAccuracyRadiusSettings].
+ */
+internal object LocationComponentAccuracyRadiusAttributeParser {
+  /**
+   * Parse [AttributeSet] to [LocationComponentSettings].
+   *
+   * @param context Context
+   * @param attrs AttributionSet
+   */
+  fun parseLocationComponentAccuracyRadiusSettings(context: Context, attrs: AttributeSet?, pixelRatio: Float = 1.0f): LocationComponentAccuracyRadiusSettings {
+    val typedArray = context.obtainStyledAttributes(attrs, R.styleable.mapbox_MapView, 0, 0)
+    try {
+      return LocationComponentAccuracyRadiusSettings(
```

Comment:
should we return some default value if smth goes wrong?

Reply:
In what scenario?

Reply:
but why you add `try` block?
for example some error in parsing color. user can set some strange value.

Reply:
Guess this try is to take the advantage of `finally` recycling the typedArray. This is the same style in all our parsers. @pengdev Please fix me if I'm wrong.

Reply:
If exception is not possible here at all - we could just use `.also{}` block, try finally signals that we _could_ face an exception

Reply:
Created a ticket to track this.